lua/telescope/pickers/entry_display.lua

```diff
@@ -68,7 +68,7 @@ entry_display.create = function(configuration)
 
   return function(self, picker)
     local results = {}
-    for k, v in ipairs(self) do
+    for k, v in pairs(self) do
```

Comment:
e.g. when `hide_filename` is set i would input a `nil` at index 1 and index 1 is not in the table. So the table looks like this:
```lua
{
  [2] = entry.tag,
  [3] = line
}
```
ipairs will not run if the table does not start with 1.

Reply:
We could also do something like this.
```lua
for i=1, #self do
  if self[i] ~= nil then
    table.insert(results, generator[i](self[i], picker))
  end
end
```
I think this should be better

Reply:
I think this is better. I think `table.getn` or something is the better way for this as well, rather than `#`? I can't remember off-hand.

Reply:
Wait i didn't even test my other solution, I get 0 as result of the length for both `#` and `table.getn` if the table is not numbered. I will take another look at this tomorrow, maybe i miss something here.